ligo/test/Test/Ligo/BaseDAO/Proposal.hs

```diff
@@ -0,0 +1,19 @@
+-- SPDX-FileCopyrightText: 2020 TQ Tezos
+-- SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+module Test.Ligo.BaseDAO.Proposal
+  ( test_BaseDAO_Proposal
+  ) where
+
+import Universum hiding (compare, drop, (>>))
+
+import Test.Tasty (TestTree)
+
+import BaseDAO.ShareTest.Proposal
+import Test.Ligo.BaseDAO.Common
+
+{-# ANN module ("HLint: ignore Reduce duplication" :: Text) #-}
```

Comment:
```suggestion
import Test.Tasty (TestTree)

import BaseDAO.ShareTest.Proposal
import Test.Ligo.BaseDAO.Common
```
It seems both the duplication ignoring and the universum import are unnecessary here.